source/user-manual/capabilities/vulnerability-detection/how_it_works.rst

```diff
@@ -17,31 +17,42 @@ The global vulnerability database is created automatically, currently pulling da
-- Partial scans: Only new packages are scanned and the vulnerabilities detected are alerted.
+For more details we have then three different scan types that are described below.
+
+- `Baseline`: This scan type will be triggered the first time after `Vulnerability Detector` is enabled. It performs a full scan for every single package installed as well as the operating system. As a result, the CVEs inventory will have the information of the vulnerabilities detected but no alerts will be generated.
+- `Full scan`: In this scan type, every single package installed as well as the operating system are scanned. It runs only when the configured :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` expires and the CVEs database is updated with new information. As a result, the newly detected vulnerabilities and the ones that no longer affect the agent will be alerted.
+- `Partial scan`: Only new packages are scanned and the new vulnerabilities detected are alerted together with deleted or updated packages.
 
 There are few considerations that arise from this behavior:
 
-- The user can not trigger a full scan manually. The only option is to disable and enable Vulnerability Detector to trigger the baseline scan. For that purpose, a service restart is needed.
-- The partial scans generate alerts for new packages, but they do not delete alerts for removed packages.
-- Partial scans can be triggered with a Manager restart.
+- The user can not trigger a `Full scan` manually. The only option is to disable and enable `Vulnerability Detector` to trigger the `Baseline` scan. For that purpose, a service restart is needed.
+- The `partial scan` can be triggered with a manager restart and generate alerts for new packages, as well as alerts for removed packages.
+- The new :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` setting, will protect the manager performance from running too often `Full scans` in case of receiving many updates of vulnerabilities feeds.
+
+Alerts
+^^^^^^
+
+- No alerts are generated for the results of a `Baseline` scan.
+- Detection alerts are generated in both, `Partial scan` and `Full scan` when is detected that a vulnerability affects one of the packages scanned.
+- Removal alerts are generated in both, `Partial scan` and `Full scan`, when is detected that a vulnerability is obsolete.
 
 Check :doc:`Vulnerability detector settings<../../reference/ossec-conf/vuln-detector>` for more configuration details.
 
-The following example may be useful to understand all steps involved
+The following example diagram may be useful to understand all steps involved in the `vulnerability detector` workflow.
```

Comment:
I think we can express this like
`The following example diagram may be useful to understand the steps involved in a typical `vulnerability detector` workflow.`
to reflect that this is only a simplified workflow diagram.

Reply:
Fixed!